policies/tagging/policy.json

```diff
@@ -6,7 +6,13 @@
       "parameters": {
         "excludedResourceTypes": {
           "defaultValue": [
-            "Microsoft.Network/networkWatchers"
+            "Microsoft.Network/networkWatchers",
+            "Microsoft.Web/serverfarms",
+            "Microsoft.Web/sites",
+            "Microsoft.Web/connections",
+            "microsoft.operationalinsights/workspaces",
```

Comment:
I believe this is log analytics, pretty sure it can be tagged?

![image](https://user-images.githubusercontent.com/21194782/131854842-3ec6d781-8e37-4562-b398-698b4bc15087.png)